buildtest/cli/config.py

```diff
@@ -153,6 +155,9 @@ def view_configuration(configuration, theme=None, pager=None):
     console.rule(configuration.file)
     console.print(syntax)
 
+def view_profiles(configuration, theme=None, pager=None):
+    """Display the list of profiles for buildtest configuration file.This implements command ``buildtest config profiles list`
+    """
```

Comment:
you have no code for this method.

Reply:
If this prints the requested information needed from the feature then it is correct. If not, you need to probably add code for this method. Not sure myself.